polkadot/xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -328,6 +343,25 @@ impl<Network: Get<Option<NetworkId>>, AccountId: From<[u8; 32]> + Into<[u8; 32]>
 	}
 }
 
+/// Extracts the `AccountId32` from the passed Treasury plurality if the network matches.
```

Comment:
```suggestion
/// Returns specified `TreasuryAccount` as `AccountId32` if passed `location` matches Treasury plurality.
```

I dont see any network matching here, I assume this is kind of copy/paste from `AccountId32Aliases` :)